ilastik/widgets/stackFileSelectionWidget.py

```diff
@@ -207,46 +209,50 @@ def _getGlobString(self, directory):
         return os.path.pathsep.join(globstrings)
 
     @staticmethod
-    def _findCommonInternal(h5Files):
-        """Tries to find common internal path (containing data)
+    def _h5N5FindCommonInternal(h5N5Files):
+        """
+        Tries to find common internal path (containing data)
 
         Method is used, when a directory is selected and the internal path is,
         thus, unclear.
 
         Args:
-            h5Files (list of strings): h5 files to be globbed internally
+            h5Files or hń5Files (list of strings): h5 or n5 files to be globbed internally
 
         Returns:
             list of internal paths
         """
-        h5 = h5py.File(h5Files[0], mode='r')
-        internal_paths = set([x['name'] for x in lsHdf5(h5, minShape=2)])
+        h5 = OpStreamingH5N5Reader.get_h5_n5_file(h5N5Files[0], mode='r')
+        internal_paths = set([x['name'] for x in lsH5N5(h5, minShape=2)])
         h5.close()
-        for h5File in h5Files[1::]:
-            h5 = h5py.File(h5File, 'r')
+        for h5N5File in h5N5Files[1::]:
+            h5 = OpStreamingH5N5Reader.get_h5_n5_file(h5N5File, 'r')
             # get all files with with at least 2D shape
-            tmp = set([x['name'] for x in lsHdf5(h5, minShape=2)])
+            tmp = set([x['name'] for x in lsH5N5(h5, minShape=2)])
```

Comment:
`maxshape = 5` was implied by the default argument if `lsHdf5`, I guess - so it was, in fact used.